Source/WebCore/css/parser/CSSParserTokenRange.h

```diff
@@ -85,13 +85,13 @@ class CSSParserTokenRange {
 
     void consumeComponentValue();
 
-    void consumeWhitespace()
+    void consumeWhitespaceLikeTokens()
```

Comment:
This set of renames seems unnecessary. I understand that the whitespace token now doesn’t include newlines. But I think that this function and the related ones can include both whitespace and newline without a name change. For example, `isASCIIWhitespace` includes newlines. Just because we are using a separate token for newlines that doesn’t mean that newlines have become “not whitespace“. If we want the naming to be perfectly consistent, I suggest renaming `WhitespaceToken` to `NonNewlineWhitespaceToken`.

This pull request will be a lot smaller if we don’t insist on referring to whitespace including newlines by a new name, and there is no compelling reason to that.